app/views/sales/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<pre><%= params %></pre>
+<h1> <%= @vendor.name %> </h1>
+
+<table class="table table-striped" class="table table-hover">
+    <tr>
+      <th class="headings" colspan=2>SALES </th>
+    </tr>
+    <tr class="header-row">
+	   <th>Amount</th>
+     <th>Purchase Time</th>
+    </tr>
+    <tr class="row">
+	    <td><%= sale.amount %></td>
+      <td><%= sale.purchase_time %></td>
+      <td><%= link_to 'Show', vendor_sale_path(@vendor, sale) %></td>
+      <td><%= link_to 'Destroy', [@vendor, sale], confirm: 'Are you sure?', method: :delete %></td>
+    <% end %>
```

Comment:
Hey! so I couldn't actually access this page from the Heroku site (even you have a route for it), and  I can't see how page will show, since the sale variable is not define anywhere. I feel like you're missing the ".each" loop to get an access to sale.